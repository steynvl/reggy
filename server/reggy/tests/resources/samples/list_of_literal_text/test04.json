{
  "sampleStringsInfo": [
    [
      {
        "markerType": "List of literal text",
        "markedStrings": [
          "adsad"
        ],
        "markerInfo": {
          "literalText": [
            "abc",
            "aerfg"
          ],
          "matchAnythingExceptSpecified": true,
          "caseInsensitive": false
        },
        "repeatInfo": {
          "repeat": "2"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
