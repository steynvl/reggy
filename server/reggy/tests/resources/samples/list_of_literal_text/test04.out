Java
Pattern regex = Pattern.compile("(?!a(?:bc|erfg)).{2}");
(?!a(?:bc|erfg)).{2}
Perl
my $regex = /(?!a(?:bc|erfg)).{2}/;
(?!a(?:bc|erfg)).{2}
POSIX
(?!a(?:bc|erfg)).{2}
(?!a(?:bc|erfg)).{2}
Python
regex = re.compile(r'(?!a(?:bc|erfg)).{2}')
(?!a(?:bc|erfg)).{2}
JavaScript
const regex = /(?!a(?:bc|erfg)).{2}/;
(?!a(?:bc|erfg)).{2}
PHP
$regex = '/(?!a(?:bc|erfg)).{2}/';
(?!a(?:bc|erfg)).{2}
Golang
regex, _ := regexp.Compile("(?!a(?:bc|erfg)).{2}")
(?!a(?:bc|erfg)).{2}
Rust
let re = Regex::new(r"(?!a(?:bc|erfg)).{2}").unwrap();
(?!a(?:bc|erfg)).{2}
C#
var re = new Regex(@"(?!a(?:bc|erfg)).{2}");
(?!a(?:bc|erfg)).{2}
Scala
val re = "(?!a(?:bc|erfg)).{2}".r
(?!a(?:bc|erfg)).{2}
Kotlin
val regex = Regex("(?!a(?:bc|erfg)).{2}")
(?!a(?:bc|erfg)).{2}